src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/Input.js

```diff
@@ -15,6 +15,7 @@ export default class Input<T: ?string | ?number> extends React.PureComponent<Inp
     static defaultProps = {
         alignment: 'left',
         collapsed: false,
+        disabled: false,
```

Comment:
Why don't we set the `disabled` prop like this on every other component as well? Then we wouldn't have to make the `disabled` prop optional, and can be sure that it is always `false` or `true`, and never `undefined`, which would probably make it easier to reason about stuff.